docs/data/data-grid/row-grouping/row-grouping.md

```diff
@@ -91,6 +91,15 @@ For instance if you group by `"director"`, you will have two columns titled _Dir
 To automatically hide the grouped columns, use the `useKeepGroupedColumnsHidden` utility hook.
 The hook will automatically hide the columns when added to the model and show them back when removed from it.
 
+:::warning
+This hook is not compatible with the deprecated column property `hide`.
+
+To manage column visibility with `columnVisibilityModel`, `initialState`, or both together, you must pass them to hook parameters.
+:::
+
+Bellow are two examples about how to use `columnVisibilityModel`, or `initialState` with `useKeepGroupedColumnsHidden` hook.
+You can mix the two examples to support both at the same time.
```

Comment:
In which scenario would you use both an initial state and a control state ? The control state would totally override the initial state right ?

Reply:
Oh ok, I thought you were talking about setting the same model in both control and initial mode